### ISSUE TITLE
Radio Button Group breaks when radio value is an integer

### ISSUE THREAD

**Issue by bengfarrell** [adobe/spectrum-web-components#364]
### Expected Behaviour
No JS error

#### Sample Code that illustrates the problem

```
<sp-radio-group selected="${Model.input.outputStride}" name="outputstride">
          <sp-radio value="8">8</sp-radio>
          <sp-radio value="16">16</sp-radio>
</sp-radio-group>
```

#### Logs taken while reproducing problem

radio-group.ts:46 Uncaught DOMException: Failed to execute 'querySelector' on 'Element': 'sp-radio[value=16]' is not a valid selector.
    at HTMLElement.set selected [as selected] (https://localhost:8080/node_modules/@spectrum-web-components/radio-group/lib/radio-group.js:34:20)

The error boils down to the following code, where sp-radio[value=xxxx] is query selected.

```
    public set selected(value: string) {
        const radio = value
            ? (this.querySelector(`sp-radio[value=${value}]`) as Radio)
            : undefined;
```

In my local environment, wrapping in quotes seems to fix the issue, but I don't know if there are 
larger implications to doing so. I'm about to extend to fix it on my end so I can continue on with my work:

`? (this.querySelector(`sp-radio[value="${value}]"`) as Radio)`

**Comment by Westbrook** [comment]
@bengfarrell nice catch, that's a tricky situation! I don't see any way that would break anything from the outside. It also adds some added stability in the case that someone sets a value of a string with spaces in it `radio value`, etc.

Would you be able to/willing to submit a PR with that change?

**Comment by bengfarrell** [comment]
Sure thing, done!

**Comment by Westbrook** [comment]
fixed by #378
